src/mjolnir/landmark_builder.cc

```diff
@@ -95,7 +97,7 @@ struct LandmarkDatabase::db_pimpl {
     if (flags & SQLITE_OPEN_CREATE) {
       // make the table
       const char* table =
-          "SELECT InitSpatialMetaData(1); CREATE TABLE IF NOT EXISTS landmarks (name TEXT, type TEXT)";
+          "SELECT InitSpatialMetaData(1); CREATE TABLE IF NOT EXISTS landmarks (id INTEGER PRIMARY KEY, name TEXT, type TEXT)";
```

Comment:
i wonder if we need to create an index on the primary key. i read somewhere on the internet that if you declare a primary key in the way that you have here you won't because under the hood sqlite treats it as an alias of the `rowid` (ordinal row number) for the table.

Reply:
I searched online and it seems SQLite automatically does indexing for primary key.
https://stackoverflow.com/questions/3379292/is-an-index-needed-for-a-primary-key-in-sqlite